eth2/types/src/indexed_attestation.rs

```diff
@@ -0,0 +1,136 @@
+use crate::{test_utils::TestRandom, AggregateSignature, AttestationData, Bitfield};
+use serde_derive::{Deserialize, Serialize};
+use ssz_derive::{Decode, Encode};
+use test_random_derive::TestRandom;
+use tree_hash::TreeHash;
+use tree_hash_derive::{CachedTreeHash, SignedRoot, TreeHash};
+
+/// Details an attestation that can be slashable.
+///
+/// To be included in an `AttesterSlashing`.
+///
+/// Spec v0.6.0
+#[derive(
+    Debug,
+    PartialEq,
+    Clone,
+    Serialize,
+    Deserialize,
+    Encode,
+    Decode,
+    TreeHash,
+    CachedTreeHash,
+    TestRandom,
+    SignedRoot,
+)]
+pub struct IndexedAttestation {
+    /// Lists validator registry indices, not committee indices.
+    pub custody_bit_0_indices: Vec<u64>,
+    pub custody_bit_1_indices: Vec<u64>,
+    pub data: AttestationData,
+    pub custody_bitfield: Bitfield,
```

Comment:
I think `custody_bitfield` doesn't belong here anymore :)

https://github.com/ethereum/eth2.0-specs/blob/v0.6.1/specs/core/0_beacon-chain.md#indexedattestation

Reply:
Additionally, I noticed we're missing the graffiti field on `BeaconBlockBody`. I couldn't find the code in this diff.